cancer_ai/validator/models.py

```diff
@@ -1,5 +1,15 @@
-from typing import List
-from pydantic import BaseModel
+from typing import List, ClassVar, Optional
+from pydantic import BaseModel, EmailStr, Field, ValidationError
+from datetime import datetime
+import yaml
+
+class SingletonMeta(type):
```

Comment:
what does it do? it seems not used by anything.

simplest singleton
```
class Foo(object):
     pass

some_global_variable = Foo()
```

Reply:
Right, I forgot to remove it. It was used previously to create singleton